settings.gradle

```diff
@@ -1,3 +1,18 @@
+pluginManagement {
+  resolutionStrategy {
+    eachPlugin {
+      String id = requested.id.id
+      if (id.startsWith("com.android")) {
+        useModule("com.android.tools.build:gradle:4.0.0")
+      } else if (id.startsWith("org.jetbrains.kotlin")) {
+        String kotlinVersion = hasProperty('square.kotlinVersion') ?
+            getProperty('square.kotlinVersion') : '1.3.72'
+        useModule("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlinVersion")
+      }
+    }
+  }
+}
```

Comment:
This is necessary to use AGP and kotlin in the `plugins {}` format :/. Kotlin does work without this in the kotlin gradle DSL at least

Reply:
Are you sure? I thought the root `build.gradle` handles this, too. I tested it locally first and it wasn't necessary for me.

Reply:
...ok I don't know how it works without it but I just tried and you're right. AGP doesn't publish plugin markers as far as I know, so this is some magic

Reply:
4cc09ce